doc/sphinx/proof-engine/ltac.rst

```diff
@@ -3,6 +3,21 @@
 Ltac
 ====
 
+.. note::
+
+   As of today, writing automation using Ltac is discouraged.
```

Comment:
Well, it will keep being discouraged "as of today" for every day in the future until Ltac1 is removed, and then this note will be removed as well. Do you have a suggestion to convey this idea differently?

Reply:
```suggestion
   Writing automation using Ltac is discouraged.
```